e2e/cypress/tests/integration/collapsed_reply_threads/global_threads_spec.js

```diff
@@ -0,0 +1,330 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @collapsed_reply_threads
+
+import TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Collapsed Reply Threads', () => {
+    let testTeam;
+    let testChannel;
+    let user1;
+    let user2;
+    let user3;
+    let rootPost;
+    let replyPost1;
+    let replyPost2;
+
+    const messages = {
+        ROOT: 'ROOT POST',
+        REPLY1: 'REPLY 1',
+        REPLY2: 'REPLY 2',
+    };
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                ThreadAutoFollow: true,
+                CollapsedThreads: 'default_off',
+            },
+        });
+
+        // # Create new channel and other user, and add other user to channel
+        cy.apiInitSetup({loginAfter: true, promoteNewUserAsAdmin: true}).then(({team, channel, user}) => {
+            testTeam = team;
+            user1 = user;
+            testChannel = channel;
+
+            // # Visit the channel
+            cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+            // # close the trial modal
+            cy.get('#startTrialModal', {timeout: TIMEOUTS.HALF_MIN}).find('.modal-header button.close').first().click();
+
+            cy.apiSaveCRTPreference(user1.id, 'on');
+            cy.apiCreateUser({prefix: 'user2'}).then(({user: newUser}) => {
+                user2 = newUser;
+
+                cy.apiAddUserToTeam(testTeam.id, user2.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, user2.id);
+                });
+            });
+
+            cy.apiCreateUser({prefix: 'user3'}).then(({user: newUser}) => {
+                user3 = newUser;
+
+                cy.apiAddUserToTeam(testTeam.id, user3.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, user3.id);
+                });
+            });
+        });
+    });
+
+    beforeEach(() => {
+        // # Visit the channel
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+
+        // # Post a message as other user
+        cy.postMessageAs({sender: user1, message: messages.ROOT, channelId: testChannel.id}).then((post) => {
+            rootPost = post;
+        });
+    });
+
+    it('MM-T4379: Display: Click to open threads)', () => {
```

Comment:
Should be fine but only for consistency. Same with others.
```suggestion
    it('MM-T4379 Display: Click to open threads)', () => {
```